functions/Utils.js

```diff
@@ -1,21 +1,127 @@
+const admin = require('firebase-admin');
+admin.initializeApp();
+// const sgMail = require('@sendgrid/mail');
+// const sgMail = require('@sendgrid/mail');
 // Push the given notification obj to account.notifications
-var pushNotification = function (label, acctRef, notification) {
+const functions = require('firebase-functions');
+const nodemailer = require('nodemailer');
+
+
+var pushNotification = function (label, acctRef, notification, templateId) {
     let promise = acctRef.child('notifications').push(notification);
+
     console.info('Notified ' + label + ' "' + acctRef.key + '": ', notification);
+    var path = ('accounts/' + acctRef.key + '/email');
+    // If it is a donating agency, have to first get the primary contact from firebase
+    if(label === 'DA') {
+        primaryContactPath = ('donating_agencies/' + acctRef.key + '/primaryContact');
+        // Get the primary contact account ID from firebase
+        admin.database().ref(primaryContactPath).once('value').then(snapshot => {
+            // Set path to primary contact email
+            path = ('accounts/' + snapshot.val() + '/email');
+            // Send an email with updated path
+            admin.database().ref(path).once('value').then(snapshot => {
+                return emailNotification(snapshot.val(), templateId);
+            })
+                .catch(err => {
```

Comment:
The indentation looks a little off to me here, I would put it on the same line as the end of the .then, so it would look like:
                }).catch(err => {
                    console.log(err);
                });

Reply:
If i try changing up the indentation it throws me an error and I'm unable to deploy the functions